### PR TITLE
tests: add sw.js to devtools tests

### PR DESCRIPTION
**Pull request by connorjclark**
Queuing this up for whenever #13396 is fixed.

### CODE REVIEW

File: third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt [hunk 1]
```diff
@@ -434,7 +438,7 @@ duplicate-id-aria: notApplicable
 duplicate-tags: notApplicable
 duplicated-javascript: numeric
 efficient-animated-content: flaky
-errors-in-console: pass
+errors-in-console: fail
```
Comment (paulirish) [inline third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt:441]: why?

File: third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Tests that audits panel works.
 
+error: Inspected target navigated or closed
```
Comment (paulirish) [inline third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt:3]: it's a bit surprising to see these error states in a "successful" run. do you know whats going on here?

**Review by paulirish — COMMENTED** [review]
this "Inspected target navigated or closed" string comes from devtools target handling?

File: third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt (hunk 2 above)
Comment (connorjclark) [reply third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt:3]: this pr aint ready look away 👀

File: third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/resources/lighthouse-basic.html [hunk 3]
```diff
@@ -2,6 +2,9 @@
 <head>
   <meta charset="UTF-8">
   <link rel="stylesheet" href="lighthouse-basic.css">
+  <script>
+    navigator.serviceWorker.register('.sw.js');
```
Comment (connorjclark) [inline third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/resources/lighthouse-basic.html:6]: `.sw.js` sus :P

**Comment by connorjclark** [comment]
xtra twist on adding a service worker: it does not hang in chromium try bots (ubunutu) but it does on GHA and my machine (mac) 😞  it's only running on mac in GHA because that's the only place our "build devtools+chromium" works (or maybe it was just the layout tests that was mac only and building is fine?)

**Comment by connorjclark** [comment]
well, we killed layout tests with fire, so, goodbye

**Pull request closed without merging**